packages/components/src/base-control/style.scss

```diff
@@ -17,6 +17,8 @@
 
 	.components-base-control__help {
 		margin-top: -$grid-unit-10;
-		font-style: italic;
+		font-size: $default-font-size - 1px;
```

Comment:
I'd create a new $helptext-font-size and reference that directly.

Reply:
Oh, that's a great idea, spinning up those changes now! Thanks for review.